source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -94,30 +193,56 @@
 
     var Template = function (data) {
         this.items = ko.observableArray();
+        this.groups = ko.observableArray();
+        this.groupedItems = ko.computed(function () {
```

Comment:
The same here, I think that it is easier to understand if all this logic in on parsing data.